db/migrate/20210811133939_add_manual_contact_attempt_history_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# This migration was created to change manual contact attempt history items from 'Contact Attempt' to 'Manual Contact Attempt' History type
+class AddManualContactAttemptHistoryType < ActiveRecord::Migration[6.1]
+  def up
+    History.where('history_type = ? AND comment like ?', 'Contact Attempt', '%contact attempt. Note:%').update_all(history_type: 'Manual Contact Attempt')
```

Comment:
The `where` query here should be based on `created_by` rather than `comment` (my fault, I think I originally gave the green light on using `comment`). In reality `comment` will work fine in production, because we only ever give manual contact attempt history items a comment with the matching string. But using `created_by` would be more robust as it is a less arbitrary field than `comment`. Plus there is the fact that the migration results in incorrect behavior when we migrate data generated by our demo script _prior_ to this commit, i.e. before [this line](https://github.com/SaraAlert/SaraAlert/pull/1040/files#diff-9bfc38ef71b77112eb91f77e35668d17dfea639c79e19de03900b201ad44b8efL950) is changed (since system contact attempts will also match the comment string).

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1040/commits/1ad2f7706728a6d00a708acf8d97f9c47b0fdad8